components/ConsolidatedFiles.tsx

```
@@ -0,0 +1,190 @@
+import { Button } from '@/components/ui/button';
+import { FileText, Download } from 'lucide-react'; // Keep Download, remove Eye, FileText will be replaced by Download icon later
+
+// Interface for the processed file data
+interface ProcessedFile {
+  baseName: string;
+  hasMd: boolean;
+  hasJson: boolean;
+}
+
+// No props needed anymore
+interface ConsolidatedFilesProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)